[ci skip] ***NO_CI*** Set home and doc_url to readthedocs page and dev_url to GitHub repo

weiji14 opened the pull request:
Documentation page at https://zen3geo.readthedocs.io. GitHub repo at https://github.com/weiji14/zen3geo.

Xref https://github.com/conda-forge/staged-recipes/pull/20198#discussion_r966084111

References:
- https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

weiji14 committed:
Set homepage of zen3geo to readthedocs

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

weiji14 commented:
https://github.com/conda-forge/staged-recipes/pull/20198#discussion_r966084111

weiji14 commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.22.0, conda-smithy 3.21.1, and conda-forge-pinning 2022.09.08.13.58.00

weiji14 committed:
[ci skip] Set doc_url and dev_url

Optional but nice to have as per https://github.com/conda-forge/staged-recipes/blob/a706a5b4099ad46e0a3d669bd6a60bb178cd7296/recipes/example/meta.yaml#L86-L88

weiji14 commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.22.0, conda-smithy 3.21.1, and conda-forge-pinning 2022.09.08.13.58.00
